docusaurus/docs/Android/01-basics/03-getting-started.md

```diff
@@ -84,10 +84,12 @@ To learn about how to create a token and different user types, see [Tokens & Aut
 
 If the `connectUser` call was successful, you are now ready to use the SDK! 🎉
 
-The methods of the `ChatClient` class allow you to create channels, send messages, add reactions, and perform many more low-level operations. You can also use the SDK's pre-built UI Components that come with ViewModels that will perform data fetching and sending for you, as described below.
+The methods of the `ChatClient` class allow you to create channels, send messages, add reactions, and perform many more low-level operations. You can also use the SDK's pre-built UI Components that will perform data fetching and sending for you, as described below.
 
 ### Adding UI Components
 
-To add the UI Components to your application, see the [Get Started](../03-ui/01-overview.md) page of the UI Components section first, which explains how the components work.
+There are two UI Component implementations available: one built on regular old Android Views that you can use in XML layouts, and another built from the ground up in [Jetpack Compose](https://developer.android.com/jetpack/compose).
```

Comment:
> old Android Views that you can use in XML layouts

Isn't it possible to use Jetpack Compose in XML layouts?
I would rephrase this sentence a little bit: `There are two UI Component implementations available: one built on regular old, XML based Android Views,`

Reply:
Good call, updated this wording